packages/@aws-cdk/aws-cognito/lib/user-pool.ts

```diff
@@ -0,0 +1,184 @@
+import lambda = require('@aws-cdk/aws-lambda');
+import cdk = require('@aws-cdk/cdk');
+import { CfnUserPool } from './cognito.generated';
+
+/**
+ * Standard attributes
+ */
+export enum Attribute {
```

Comment:
`Attribute` feels very generic a name to me. Maybe that's okay, but maybe `UserPoolAttribute` is better?